sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/InputForbidFieldAccessCheckStaticTest.java

```diff
@@ -0,0 +1,18 @@
+package com.github.sevntu.checkstyle.checks.coding;
+
+import com.puppycrawl.tools.checkstyle.api.LocalizedMessage;
+
+import static java.util.Locale.ROOT;
+import static java.lang.Integer.MAX_VALUE;
+
+public class InputForbidFieldAccessCheckStaticTest {
+
+    public static void setLocaleToRoot() {
+        LocalizedMessage.setLocale(ROOT);
```

Comment:
Another example has to be added in another file when using the fully qualified class name and no import. It should not conflict with any similar named classes in the file.